include/triton/common/logging.h

```diff
@@ -73,9 +73,25 @@ class Logger {
   // Get the logging format.
   Format LogFormat() { return format_; }
 
+  // Get the logging format as a string.
+  std::string LogFormatString()
+  {
+    std::string format = "default";
+    if (format_ == Format::kISO8601) {
+      format = "ISO8601";
+    }
+    return format;
+  }
+
   // Set the logging format.
   void SetLogFormat(Format format) { format_ = format; }
 
+  // Get the log output file name.
+  const std::string& GetLogOutFile() { return filename_; }
+
+  // Set the log output file.
+  void SetLogOutFile(const std::string& filename) { filename_ = filename; file_name_changed_=true; }
```

Comment:
Is the code formatted? This line should be broken down to multiple lines.

Reply:
Do we need to lock the mutex here in case multiple clients are changing the logging simultaneously?

Reply:
I can add a lock here, however, does that mean we should be adding a lock for every function where we change attributes?